.travis.yml

```diff
@@ -4,15 +4,25 @@ language: python
 matrix:
   include:
     - python: 3.6
-      env: TOXENV=py,codecov
+      env: TOXENV=py36-sqla1.2,codecov
```

Comment:
@ticosax you can keep the same number of python versions as before and substitute the previous `env` with `TOXENV=py-sqla{1.2,1.3}, codecov `

Reply:
thanks! that's what I was looking for when I first understood the feedback.
But today I'm not sure that's what the maintainers are after anymore. I'll probably go into a direction on testing sqalchemy 1.2 and 1.3 only for python3.6 instead of all supported python versions, as it seems preferred by the maintainers.